api/specs/web-server/_users.py

```diff
@@ -45,6 +50,25 @@ async def update_my_profile(_profile: ProfileUpdate):
     ...
 
 
+@router.get(
+    "/me/preferences",
+    response_model=Envelope[FrontendUserPreferencesGet],
+)
+async def get_user_preferences():
+    ...
+
+
+@router.patch(
+    "/me/preferences/{frontend_preference_name}",
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+async def set_frontend_preference(
+    _params: Annotated[FrontendUserPreferencePatchPathParams, Depends()],
```

Comment:
I think it is unnecessary to make variables so long in such a **well defined context**